scripts/dependencies/tools.ts

```diff
@@ -558,3 +558,26 @@ export class ECRCredHelper implements Dependency, GitHubDependency {
     return semver.rcompare(version1, version2);
   }
 }
+
+export class SpinShim implements Dependency {
```

Comment:
Should this implement `GitHubDependency` too?

Reply:
Yes. Updated; downgraded shim versions to 0.10.0; amended commit.